### ISSUE TITLE
Fixing the enroll option

### ISSUE THREAD

**Issue by harshita0802goswami** [SahityaRoy/E-Learning-freesite#11]
Aligning the enroll option in the C++ basic-advanced course

**Comment by SahityaRoy** [comment]
@harshita0802goswami  yeah please you can adjust the enroll button also for mobile user

**Comment by harshita0802goswami** [comment]
![image](https://user-images.githubusercontent.com/68657465/123915290-f8ab4980-d99d-11eb-8855-8ff663414475.png)
For desktop!

**Comment by SahityaRoy** [comment]
yes @harshita0802goswami  please go ahed
